setup.py

```diff
@@ -1,55 +1,14 @@
 #!/usr/bin/env python
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
-# -*- encoding: utf-8 -*-
 
+import sys
+import setuptools
+from distutils.version import LooseVersion
 from setuptools import setup
 
+# Setuptools 30.3.0 or later is needed for setup.cfg options to be used
+if LooseVersion(setuptools.__version__) < LooseVersion('30.3.0'):
```

Comment:
Why not put this in `pyproject.toml`?

Reply:
This is in case people call setup.py directly

Reply:
Hmm. Okie dokie then. Merging.